docker/README.md

```diff
@@ -1,30 +1,45 @@
 ## Step 1. Install docker
+
 Review the instructions for installation docker [here](https://docs.docker.com/engine/install/ubuntu/) and configure HTTP or HTTPS proxy behavior as [here](https://docs.docker.com/config/daemon/systemd/).
 
 ## Step 2. Install nvidia-docker
 
 *Skip this step if you don't have GPU.*
 
-Review the instructions for installation docker [here](https://github.com/NVIDIA/nvidia-docker)
+Review the instructions for installation docker [here](https://github.com/NVIDIA/nvidia-docker).
 
 ## Step 3. Build image
+
 In the project folder run in terminal:
- ```
- sudo docker image build --network=host --build-arg http_proxy=http://example.com:80 --build-arg https_proxy=http://example.com:81 --build-arg 
-ftp_proxy=http://example.com:80 <PATH_TO_DIR_WITH_DOCKERFILE>
- ```
+```
+sudo docker image build \
+--network=host \
+--build-arg http_proxy=http://example.com:80 \
+--build-arg https_proxy=http://example.com:81 \
+--build-arg ftp_proxy=http://example.com:80 \
+<PATH_TO_DIR_WITH_DOCKERFILE>
+```
+
+Use `--http_proxy` , `--https_proxy`, `--ftp_proxy`, `--network` to duplicate the network settings of your localhost into context build.
```

Comment:
Better way to setup proxy https://docs.docker.com/network/proxy/#configure-the-docker-client.

Reply:
Thanks for your comment! I agree with you. I will try this approach.

Reply:
It works.